admin-auth/src/test/kotlin/com/epam/drill/admin/auth/UserRepositoryImplTest.kt

```diff
@@ -115,6 +139,14 @@ class UserRepositoryImplTest {
         }
     }
 
+    @Test
+    fun `after database migration 2 default users must be inserted`() = withTransaction {
```

Comment:
-`migration 2` definition is a bit vague - change for description of "what" that migration does
-I don't see "arrange/act" here, only assertions - where are the actual preparations happening?
_tbd_

Reply:
Changed the title to `after database migration all default users must be inserted`